app/javascript/src/components/Dashboard/Notes/NoteCard.jsx

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable no-console */
+/* eslint-disable no-undef */
+// @ts-nocheck
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-duplicate-imports */
+import React from "react";
+
+// @ts-ignore
+// @ts-ignore
+import { Clock } from "@bigbinary/neeto-icons";
+// @ts-ignore
+import { Typography, Tag, Tooltip } from "@bigbinary/neetoui";
+// @ts-ignore
+// @ts-ignore
+import { Avatar } from "@bigbinary/neetoui";
+// @ts-ignore
+
+const NoteCard = ({ note }) => (
```

Comment:
Why do you feel that you have to prefix the `Note` keyword when the component is already within the `Notes` namespace? Is there any other type of card within this namespace?

Read: https://academy.bigbinary.com/learn-react/pages/structuring-code#:~:text=Avoid%20Unnecessary%20Namespacing